README.md

```diff
@@ -10,6 +10,24 @@
 [![Gitter](https://badges.gitter.im/jenkinsci/configuration-as-code-plugin.svg)](https://gitter.im/jenkinsci/configuration-as-code-plugin)
 
 <img src="plugin/src/main/webapp/img/logo-head.svg" width="192">
+## Documentation
+
+<a name="handling-secrets"></a>
+
+You can find more documentation about JCasC here:
+
+- [Handling Secrets](./docs/features/secrets.adoc)
+- [Exporting configurations](./docs/features/configExport.md)
+- [Validating configurations](./docs/features/jsonSchema.md)
+- [Triggering Configuration Reload](./docs/features/configurationReload.md)
+
+The configuration file format depends on the version of jenkins-core and installed plugins.
+Documentation is generated from a live instance, as well as a JSON schema you can use to validate configuration file
+with your favourite YAML tools.
+
+The JSON Schema documentation can be found [here](./docs/features/jsonSchema.md).
+
+**TODO**: Provide a Dockerfile to generate documentation from specified jenkins-core release and plugins.
 
 ## Introduction
```

Comment:
Introduction on bottom now.

Reply:
Should I keep the reference above or the introduction?

Reply:
@timja mentioned keeping reference at the top so that people find it easy

Reply:
I think if we remove some of the text around it in my last suggestion it should be okay, otherwise we might want to think about shortening the introduction.

The reason for moving these links up is they aren’t very discoverable and people don’t scroll far enough to find them, the hope is that with them at or near the top more people will find them